resources/style.css

```
@@ -7,28 +7,59 @@ body {
   height: 100vh;
 }
 
+nav.navbar {
+  background-color: rgba(52, 51, 50, 0.8);
```

I would increase the opacity to `0.9` to improve readability of the menu on mobile.

```suggestion
  background-color: rgba(52, 51, 50, 0.9);
```